src/index.js

```diff
@@ -1 +1,7 @@
-// export your functions here
+export const every = (array, test) => {
```

Comment:
```suggestion
export const every = (array, callback) => {
```
I reckon `callback` helps to identify this as a function more so than `test`

Reply:
This part is up to you, but I reckon if you're going to go with a named function expression like `const every = someFn`, you might as well just declare the function and skip the expression part:
`function every(array, callback)`.